src/Clock.tsx

```diff
@@ -0,0 +1,52 @@
+import { Component } from 'react';
+
+type Props = {
+  name: string;
+};
+
+type State = {
+  today: Date;
+};
+
+export class Clock extends Component<Props, State> {
+  state = {
+    today: new Date(),
+  };
+
+  componentDidMount() {
+    const { today } = this.state;
+
+    window.setInterval(() => {
+      this.setState({ today: new Date() });
+
+      console.info(today);
```

Comment:
```suggestion
      console.info(today.toLocaleTimeString());
```
<img width="102" alt="Снимок экрана 2022-09-05 в 21 59 03" src="https://user-images.githubusercontent.com/107410035/188502895-f6a952ae-8b9c-4a88-bcde-5e89562e046b.png">

Add localeTimeString for correct date format:)